server/helpers/reqHelper.js

```diff
@@ -3,20 +3,20 @@ const requestHelper = {
     if (data) {
       return res.status(statusCode).json(Object.assign({
```

Comment:
Airbnb styleguide prefers spread operator to Object.assign. Checkout section 3.8 of https://github.com/airbnb/javascript
Revert accordingly